src/uplift/bot/mozdata.ini

```diff
@@ -0,0 +1,2 @@
+[User-Agent]
+name = uplift-bot
```

Comment:
I'm not sure that would work as expected in our Nix environment.
Could you add a unit test that makes a Bugzilla query, using [responses](https://github.com/getsentry/responses) to catch the query and checks the user agent in the headers ?

A good example in this project is [test_report.py](https://github.com/mozilla/release-services/blob/master/src/uplift/bot/tests/test_report.py). Please create a new file in the same folder (`test_libmozdata.py` ?)

Reply:
Sure, will do that.
just now I only tested it using python shell, and `import config from libmozdata.bugzilla`
then I test to get the user agent from config using `config.get('User-Agent', 'name', 'libmozdata')` as shown here

https://github.com/mozilla/libmozdata/blob/319368f447624204a263697a772a07e3968b5339/libmozdata/connection.py#L54